paper/src/main/java/com/denizenscript/denizen/paper/PaperModule.java

```diff
@@ -63,6 +63,7 @@ public static void init() {
             ScriptEvent.registerScriptEvent(PlayerRaiseLowerItemScriptEventPaperImpl.class);
         }
         ScriptEvent.registerScriptEvent(PlayerSelectsStonecutterRecipeScriptEvent.class);
+        ScriptEvent.registerScriptEvent(PlayerLoomPatternSelectScriptEvent.class);

```

Comment:
Not in alphabetical order. I know it's not directly stated that it needs to be, but its kinda like an unspoken standard :P